src/client/Components/Util/LogTable.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+
+export default class LogTable extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      github_repo: this.props.github_repo,
+      isLoaded: false
+    };
+  }
+
+  componentDidMount() {
+    this.getComponentLogs().then();
+  }
+
+  async getComponentLogs() {
+    try {
+      const result = await fetch(`http://v9_website.ngrok.io/api/db/getComponentLogs?component=${this.state.github_repo}`,
+        {
+          method: 'GET',
+          credentials: 'include',
+          headers: {
+            Accept: 'application/json',
+            'Content-Type': 'application/json',
+            'Access-Control-Allow-Credentials': true
+          }
+        });
+      if (!result.ok) {
+        throw new Error(result.statusText);
+      }
+
+      console.log(result);
+
+      const json = await result.json();
+      console.log(json);
+      this.setState({
+        isLoaded: true,
+        componentLogs: json
+      });
+    } catch (e) {
+      console.log(e);
```

Comment:
Kinda bad UX to leave the page blank on error. Is there an issue for error handling?

Reply:
#16